lib/pengine/complex.c

```diff
@@ -570,6 +570,22 @@ unpack_requires(pe_resource_t *rsc, const char *value, bool is_default)
                  (is_default? " (default)" : ""));
 }
 
+/*!
+ * \internal
+ * \brief Parse configuration XML for a given resource
+ *
+ * Unpack the XML object containing a resource's configuration into an empty
+ * \c pe_resource_t object.
+ *
+ * \param[in]  xml_obj   XML node containing the resource's configuration
+ * \param[out] rsc       Destination for unpacking
+ * \param[...] parent    Resource's parent (\c NULL if the resource is a
+ *                       primitive)
+ * \param[in]  data_set  Cluster working set
```

Comment:
in,out following the newly established convention, since this function can change data_set members via called functions

Reply:
Ah, gotcha. I wasn't sure where we landed on that.